src/integrationtest/resources/integration-test.properties

```diff
@@ -113,6 +113,10 @@ spring.mail.properties.mail.smtp.starttls.enable: false
 #er.url: http://localhost:8080/eventrepository/search/
 er.url:
 
+######## EI Default Back-Ends
+ei.backend.instances.list.json.content=[{"name":"EI-Backend-1","host":"localhost","port":8070,"contextPath":"","https":false,"defaultBackend":true}, {"name":"EI-Backend-2","host":"localhost","port":8077,"contextPath":"","https":false,"defaultBackend":false}, {"name":"EI-Backend-3","host":"localhost","port":8078,"contextPath":"","https":false,"defaultBackend":true}]
```

Comment:
Do integration test use more then one back-end, if not the other ones may just be confusing to add.

Reply:
Removed them, they should not be here.